.gitguardian.example.yml

```diff
@@ -36,3 +36,15 @@ secret:
   # Detectors to ignore.
   ignored-detectors: # default: []
     - Generic Password
+
+iac:
+  # Exclude files and paths by globbing
+  ignored-paths:
+    - 'tests/'
+
+  # Ignore IaC vulnerability types by their id
+  ignored-policies:
+    - GG_IAC_0000
+
+  # Precise vulnerabilities minimum severity to scan for
+  minimum-severity: LOW
```

Comment:
Maybe use another value for minimum-severity.

As LOW is the lowest possible minimum severity, this parameters doesn't change anything.

Reply:
This value was suggested by the linear ticket. As the other ones, maybe it is intended to not change anything ; such as having only one policy with id `GG_IAC_0000`?

Reply:
See Henri's answer on the Linear issue